### PR TITLE
Migrated to Swift 3 syntax

### PR DESCRIPTION
**Pull request by melle**

### CODE REVIEW

**Comment by franklinyu** [comment]
Note: Swift 3 is incompatible with Swift 2.

**Comment by melle** [comment]
@franklinyu to be honest, I just needed AEXML to work in my Swift 3 project 😬 I know that there's a compatibility issue. If I understand https://github.com/tadija/AEXML/pull/70#issuecomment-224823348 correctly, the plan is to migrate master to Swift 3 when it's officially released.  I'll keep the PR updated and hope we can just merge this in September.

File: Tests/AEXMLTests.swift [hunk 1]
```diff
@@ -36,14 +36,14 @@ class AEXMLTests: XCTestCase {
...
-        return bundle.URLForResource(fileName, withExtension: withExtension)!
+        let bundle = Bundle(for: AEXMLTests.self)
+        return bundle.urlForResource(fileName, withExtension: withExtension)!
     }
     
     func xmlDocumentFromURL(url: NSURL) -> AEXMLDocument {
         var xmlDocument = AEXMLDocument()
         
-        if let data = NSData(contentsOfURL: url) {
+        if let data = NSData(contentsOf: url as URL) {
```
Comment (franklinyu) [inline Tests/AEXMLTests.swift]: `as URL` may be removed if you change the parameter type from `NSURL` to `URL`, which I think Apple encourage us to use.

**Comment by franklinyu** [comment]
Note: after this being merged, consider adding Continuous Integration. [It is said](https://www.reddit.com/r/swift/comments/434vmu/what_do_you_guys_use_for_continuous_integration/czfidnc) that Travis CI also support Swift (With Xcode 8 beta).

File: Tests/AEXMLTests.swift (hunk 1 above)
Comment (melle) [inline Tests/AEXMLTests.swift]: Ok, I updated the PR, this affects tests only.

File: Tests/AEXMLTests.swift [hunk 2]
```diff
@@ -35,12 +35,12 @@ class AEXMLTests: XCTestCase {
...
+    func URLForResource(fileName: String, withExtension: String) -> URL {
         let bundle = Bundle(for: AEXMLTests.self)
         return bundle.urlForResource(fileName, withExtension: withExtension)!
     }
     
-    func xmlDocumentFromURL(url: NSURL) -> AEXMLDocument {
+    func xmlDocumentFromURL(url: URL) -> AEXMLDocument {
         var xmlDocument = AEXMLDocument()
         
         if let data = NSData(contentsOf: url as URL) {
```
Comment (franklinyu) [inline Tests/AEXMLTests.swift]: No need to convert `URL` to `URL`.

**Comment by franklinyu** [comment]
So c03943b9d737661bdc938e1b20048bfbb435c4f8 is purely performed by the migration wizard? Seems basically fine.

**Comment by tadija** [comment]
Thanks very much for your efforts on this one. It's finally merged now!

**Pull request merged**